assembly/activities/genGlossary.js

```diff
@@ -53,12 +65,24 @@ async function getTerms(config, sourceNodes) {
   }
   return terms;
 }
-
 async function genGlossString(terms) {
-  let glossStr = `${glossFrontmatter}\n\n`;
+  let glossStr = `${glossFrontmatter}\n\n<ul id="glossary-entries">`;
```

Comment:
This doesn't necessarily need to be an ul list.
Could be h4s?
But if it is, can we remove the bullets?
I still think we should keep the tags linked and listed horizontally under the description - can style them differently though, make it smaller.
After seeing this, I don't see why we wouldn't want them there. The filter doesn't give you those details - just filters against the list.